src/helpers/screenList.ts

```diff
@@ -0,0 +1,26 @@
+import { Block } from 'core';
+import { PropsType } from 'core/Block';
+import LoginPage from 'pages/login';
+import SignupPage from 'pages/signup';
+import ProfilePage, { EditPasswordPage } from 'pages/profile';
+import ChatsPage from 'pages/chats';
+
+export enum Screens {
+  Login = 'login',
+  SignUp = 'sign-up',
+  Profile = 'setting',
```

Comment:
Поменяй пожалуйста имя этой страницы согласно ТЗ
```suggestion
  Profile = 'settings',
```